### DISCUSSION TITLE
Multiple Values Time Series Not working(?)

### DISCUSSION THREAD

**Discussion by Jrecos** [mullerpeter/databricks-grafana#87] [Q&A] [answered]
Hi Mates!

I hope you all are well!

I started with the plugin but I'm having an "issue".

I'm following the examples in the documentation, but I don't understand why the chart does not show the multiple lines as usual.

this is my query: 
![image](https://github.com/user-attachments/assets/971003f7-a39e-464d-87a9-3848b36469f6)

However, the chart is not detecting the multiline :(: 
![image](https://github.com/user-attachments/assets/7298b3f8-880d-48f6-955f-e7ae32e2ec2f)

maybe I'm doing something wrong?

this is the table view:
![image](https://github.com/user-attachments/assets/24a7ed85-465e-40d9-918d-900d9e6faa73)

Thanks for your help mates!

**Comment by mullerpeter** [comment, marked as answer]
Have you tried to enable "Long to Wide" transformation?

**Reply by Jrecos** [reply]
Hey Hi for your reply.

Effectively that was the missing thing on mi side... thanks a lot and sorry for the dummy question
